### PR TITLE
Add support for Enchantment Descriptions.

### PR DESCRIPTION
**Pull request by Darkhax**
This PR adds support for my [Enchantment Descriptions](https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions) mod and other mods with similar features.

### CODE REVIEW

File: src/main/resources/assets/requiem/lang/en_us.json [hunk 1]
```diff
@@ -185,6 +185,7 @@
   "effect.requiem.reclamation": "Reclamation",
 
   "enchantment.requiem.humanity": "Humanity",
+  "enchantment.requiem.humanity.desc": "Improves drops obtained by posessed mobs.",
```
Comment (Pyrofab) [inline src/main/resources/assets/requiem/lang/en_us.json]:
```suggestion
  "enchantment.requiem.humanity.desc": "Improves drops obtained by possessed mobs.",
```

**Review by Pyrofab — CHANGES_REQUESTED** [review]
Thank you ! Just a little typo if I am not mistaken.

**Comment by Darkhax** [comment]
Yep you're right, nice catch.

**Review by UpcraftLP — APPROVED** [review]

**Comment by Darkhax** [comment]
Hey, are there any updates on getting this merged?

**Comment by Pyrofab** [comment]
Oh yeah, thanks for reminding me

**Review by Pyrofab — APPROVED** [review]

**Comment by Darkhax** [comment]
Awesome, thank you!

**Pull request merged**